frontend/src/components/RelatedResources/TracePanel.tsx

```diff
@@ -37,7 +54,13 @@ export const TracePanel: React.FC = () => {
 							</Text>
 							<Box bb="dividerWeak" mt="12" mb="8" />
 
-							{span && <TraceSpanAttributes span={span} />}
+							{span && (
+								<TraceSpanAttributes
+									span={span}
+									query={query}
+									onSubmit={handleSubmit}
```

Comment:
could we call this `onUpdateQuery` instead of `onSubmit` to make it more clear

Reply:
Ack